tests/conftest.py

```diff
@@ -158,6 +158,26 @@ def mock_get_request(path: str, unifi_path: str, payload: dict[str, Any]) -> Non
     )
 
 
+@pytest.fixture(name="_mock_traffic_rule_endpoint")
+def endpoint_fixture(
+    mock_aioresponse: aioresponses,
+    is_unifi_os: bool,
+    traffic_rule_payload: dict[str, Any],
+) -> None:
+    """Use fixtures to mock traffic rule endpoint."""
+
+    def mock_get_request(path: str, unifi_path: str, payload: dict[str, Any]) -> None:
+        """Register HTTP response mock."""
+        url = unifi_path if is_unifi_os else path
+        data = payload
+        mock_aioresponse.get(f"https://host:8443{url}", payload=data)
```

Comment:
This should be moved into the real endpoint_fixture

Reply:
Still on my todo-list

Reply:
I get an error when adding this to _mock_endpoints when running the tests.
`ERROR tests/test_traffic_rules.py - Failed: In test_traffic_rules: function uses no argument 'traffic_rule_payload'`
I still need to investigate this further.